Userland/Libraries/LibC/crt0.cpp

```diff
@@ -39,12 +39,14 @@ int _entry(int argc, char** argv, char** env)
 
     environ = env;
     __environ_is_malloced = false;
+    __begin_atexit_locking();
 
     _init();
 
     int status = main(argc, argv, environ);
 
     exit(status);
+    exit(1);
```

Comment:
two exits?

Reply:
Aargh! Right, sorry, I added that while wrestling #10807.